owsi-core/owsi-core-examples/wicket-showcase/wicket-showcase-webapp/src/main/java/fr/openwide/core/showcase/web/application/config/spring/ShowcaseWebappApplicationPropertyConfig.java

```diff
@@ -0,0 +1,18 @@
+package fr.openwide.core.showcase.web.application.config.spring;
+
+import org.springframework.context.annotation.Configuration;
+
+import fr.openwide.core.jpa.more.business.property.service.IConfigurablePropertyService;
+import fr.openwide.core.jpa.more.config.spring.AbstractApplicationPropertyConfig;
+import fr.openwide.core.showcase.web.application.util.property.ShowcaseWebappPropertyIds;
+
+@Configuration
+public class ShowcaseWebappApplicationPropertyConfig extends AbstractApplicationPropertyConfig {
```

Comment:
Both your core property configuration and your webapp property configuration inherit from  AbstractApplicationPropertyConfig, which define singleton beans (PropertyService being one of them). Defining a singleton twice sucks. 
You should have two abstract bases for property config: one which only registers properties (which can be used anywhere, including in OWSI-Core modules and in Web applications), and the other that does that *and* defines the beans (which will only be used in the application Core module).